### PR TITLE
Tensorboard Related Documentation

### PR DESCRIPTION
**Pull request by AyeshaSanadi**
# Related Issues / Pull Requests

List all related issues and/or pull requests if there are any.

# Description

Include a brief summary of the proposed changes.

# What changes are proposed in this pull request?

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected; for instance, 
      examples in this repository need to be updated too).
- [ ] This change requires a documentation update.

# Checklist:

- [ ] My code follows the style guidelines of this project (PEP-8 with Google-style docstrings).
- [ ] My code modifies existing public API, or introduces new public API, and I updated or wrote docstrings that
      uses Google-style formatting and any other formatting that is supported by mkdocs and plugins this project
      uses.
- [ ] I have commented my code.
- [ ] My code requires documentation updates, and I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: docs/cmf_client/cmf_client.md [hunk 1]
```diff
@@ -34,15 +34,15 @@ Required Arguments
   --endpoint-url [endpoint_url]         Specify the endpoint url of minio UI.
   --access-key-id [access_key_id]       Specify Access Key Id.
   --secret-key [secret_key]             Specify Secret Key.
+  --cmf-server-url [cmf_server_url]     Specify cmf-server url. (default: http://127.0.0.1:80)
```
Comment (annmary-roy) [inline docs/cmf_client/cmf_client.md:37]: why are we documenting the server as required argument ? It should stay optional and take a default value

File: test/old_archive_code/test_init.py
Comment (annmary-roy) [inline test/old_archive_code/test_init.py:1]: why rename ?

Comment (varkha-d-sharma) [reply test/old_archive_code/test_init.py:1]: Hi Ann, this is the old code we wrote for testing. Now we have updated and elaborated version of this same code in cmf/test folder. However, didn't want to delete code. So asked Ayesha to move it to test folder.
We have moved it from cmf/examples/exampled-get-started to cmf/test/old_archieve_code.

File: docs/cmf_client/cmf_client.md (hunk 1 above)
Comment (varkha-d-sharma) [reply docs/cmf_client/cmf_client.md]: Resolved

File: docs/cmf_client/cmf_client.md [hunk 2]
```diff
@@ -275,11 +275,31 @@ Optional Arguments
...
+  -h, --help                                         show this help message and exit
+  -f [file_name],   --file_name [file_name]          Specify mlmd file name.
+  -e [exec_id],     --execution [exec_id]            Specify execution id.
+  -t [tensorboard], --tensorboard [tensorboard]      Specify path to tensorboard logs for the pipeline.
+```
+### cmf metadata export
+```
+Usage: cmf metadata export [-h] -p [pipeline_name] -j [json_file_name] -f [file_name]
+```
+`cmf metadata push` export local mlmd's metadata in json format to a json file.
```
Comment (annmary-roy) [inline docs/cmf_client/cmf_client.md:300]: Is this correct ?

Comment (AyeshaSanadi) [reply docs/cmf_client/cmf_client.md]: Hello Ann, This is not correct. Thank you for identifying the issue. I've made the necessary corrections in the latest update. Please review the changes and let me know if any further changes are required.

**Review by annmary-roy — APPROVED** [review]

**Review by annmary-roy — APPROVED** [review]

**Pull request merged**